Configuration/FlexForms/flexforms_pi1.xml

```diff
@@ -383,6 +384,7 @@
                                 <type>select</type>
                                 <renderType>selectCheckBox</renderType>
                                 <foreign_table>tx_seminars_categories</foreign_table>
+                                <foreign_table_where>AND tx_seminars_categories.pid LIKE ###PAGE_TSCONFIG_STR### ORDER BY title</foreign_table_where>
```

Comment:
Why are you using `LIKE` instead of `=` or `IN()`?

Reply:
The wildcard % workes only with LIKE

Reply:
Makes sense. Thanks!

Reply:
Ok.